### PR TITLE
Reduce bundle size by using large module inside split code chunk

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
In  PR #547 `react-phone-number-input` was moved from a lazy loaded module into the redux store. This caused a 25% bundle increase.

# Solution
There is no need to compute number validation in the redux store. My solution involves importing `react-phone-number-input`  inside the lazy loaded module.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [x] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by stefaniacardenas** [comment]
Bundle reduced by `-33.32%`

File: src/components/PhoneNumberInput/index.js [hunk 1]
```diff
@@ -18,25 +18,47 @@ const FlagComponent = ({ countryCode, flagsPath }) => (
...
+    this.state = {
+      validPredefinedNumber: false,
+    }
+  }
+
+  componentDidMount() {
+    const smsProps = this.props.sms
+    if (smsProps && smsProps.number) {
+      const validPredefinedNumber = isValidPhoneNumber(smsProps.number)
+      this.setState({validPredefinedNumber})
```
Comment (rfreitas) [inline src/components/PhoneNumberInput/index.js]: why is state necessary if the number is on the store?

Comment (rfreitas) [inline src/components/PhoneNumberInput/index.js]: why is state necessary if the number is on the store?

Comment (stefaniacardenas) [reply src/components/PhoneNumberInput/index.js]: It's not necessary

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by stefaniacardenas** [comment]
@rsilva4 It's because we import the large module inside this file `src/components/PhoneNumberInput/index.js` which is not part of the main bundle, it's loaded on demand [here](https://github.com/onfido/onfido-sdk-ui/blob/2dd18a56ba04cc9f247821f21474787efe70b785/src/components/PhoneNumberInput/Lazy.js).

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Behaviour for invalid numbers is different in this PR. If it's invalid the number doesn't show.
My opinion the old behaviour is better for the following reasons:
1. Easier to debug and integrate. If the number is invalid the error is silent, so harder to know what is going on
2. It's easier for the users to start from incorrect number, but possibly not that far off from being correct

you can test with `onfidoSdkHandle.setOptions({userDetails:{smsNumber:"+44 7721 2"}})`

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-597-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://597-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**